src/main/java/uiPages/LoginPage.java

```diff
@@ -92,25 +83,19 @@ public void open(){
     openUrl(BASE_URL + URL_PATH);
   }
 
-  public static String getExpectedLoginErrMsg() { return LOGIN_ERR_MSG; }
-
-  public static String getExpectedLoginNoUserErrMsg() { return LOGIN_NO_USER_ERR_MSG; }
-
-  public static String getExpectedLoginNoPassErrMsg() { return LOGIN_NO_PASSWORD_ERR_MSG; }
+  public static String getExpectedLoginErrMsg(String errorType) {
```

Comment:
The way these methods were implemented before it was the right way. Not sure why you changed it.
If you choose this approach then you need to make sure only the expected "errorType" is send when the getExpectedLoginErrMsg() method is call.
For example I can call the method as getExpectedLoginErrMsg("blablaerror") which is incorrect. The people who wor with this class need to know what parameter are accepted when calling this method without having to read the whole method. You can do this either by using an enum instead of String.
Anyhow the shortest and easier approach is having separate methods, as you had before.

Reply:
done